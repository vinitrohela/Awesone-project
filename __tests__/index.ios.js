import 'react-native';
import React from 'react';
import Index from '../index.ios.js';
import Button from '../index.ios';
import { shallow } from 'enzyme';
const expct = require('chai').expect
import { View } from 'react-native';
import { Text } from 'react-native';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
    <Index />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
// describe('Button', () => {
//   test('Renders', () => {
//     const component = renderer.create(
//       <Button>
//       Button
//       </Button>
//     )
//     const tree = component.toJSON()
//     expect(tree).toMatchSnapshot()
//   })
// })

describe('<Index />', () => {
  it('should be a view component', () => {
    const wrapper = shallow(<Index></Index>);
    expct(wrapper.type()).to.equal(View);
  });
  it('should have a title Welcome to React Native vinit swapnil!', () => {
    const wrapper1 = shallow(<Index/>);
    expct(wrapper1.contains(<Text>Welcome to React Native vinit swapnil!</Text>)).to.equal(true);
  });
  it('should have a title To get started, edit index.ios.js', () => {
    const wrapper1 = shallow(<Index/>);
    expct(wrapper1.contains(<Text>To get started, edit index.ios.js</Text>)).to.equal(true);
  });
  it('should have a title Press Cmd+R to reload', () => {
    const wrapper1 = shallow(<Index/>);
    expct(wrapper1.contains(<Text>Press Cmd+R to reload,{'\n'}
    Cmd+D or shake for dev menu</Text>)).to.equal(true);
  });
});
