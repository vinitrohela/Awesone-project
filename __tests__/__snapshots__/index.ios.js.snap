exports[`test renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F5FCFF",
      "flex": 1,
      "justifyContent": "center",
    }
  }>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Welcome to React Native vinit swapnil!
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    To get started, edit index.ios.js
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail">
    Press Cmd+R to reload,
    
    
    Cmd+D or shake for dev menu
  </Text>
</View>
`;
